spec/peep_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'peep'
+
+describe Peep do
+  describe '.all' do
+    it 'returns Peepdeck' do
+      PG.connect(dbname: 'chitter_app_test')
```

Comment:
I think this should you the database set in the env variable. Not sure this line is needed here anymore. Worth testing